packages/analytics-js-integrations/__tests__/integrations/Ninetailed/browser.test.js

```diff
@@ -0,0 +1,95 @@
+import Ninetailed from '../../../src/integrations/Ninetailed/browser';
+
+afterAll(() => {
+  jest.restoreAllMocks();
+});
+const destinationInfo = {
+  areTransformationsConnected: false,
+  destinationId: 'sample-destination-id',
+};
+window.ninetailed = {};
+
+describe('Ninetailed page', () => {
```

Comment:
Create an outer level `describe` integration test group and move individual API test `describe`s into it.